mappings/net/minecraft/entity/LivingEntity.mapping

```diff
@@ -32,6 +35,7 @@ CLASS net/minecraft/class_1752 net/minecraft/entity/LivingEntity
 		ARG 1 effect
 	METHOD method_7103 (Lnet/minecraft/class_1752;)V
 		ARG 1 entity
+	METHOD method_7107 getHeldStack ()Lnet/minecraft/class_2056;
```

Comment:
this doesnt feel like a yarn name getItemInHand() i believe is the correct name

Reply:
done